rust/operator-binary/src/main.rs

```diff
@@ -72,23 +77,58 @@ async fn main() -> anyhow::Result<()> {
             PodListeners::print_yaml_schema(built_info::PKG_VERSION)?;
         }
         stackable_operator::cli::Command::Run(ListenerOperatorRun {
-            tracing_target,
             csi_endpoint,
             mode,
+            telemetry_arguments,
             cluster_info_opts,
         }) => {
-            stackable_operator::logging::initialize_logging(
-                "LISTENER_OPERATOR_LOG",
-                "listener-operator",
-                tracing_target,
-            );
-            stackable_operator::utils::print_startup_string(
-                &format!("{} ({})", crate_description!(), mode.as_ref()),
-                crate_version!(),
-                built_info::GIT_VERSION,
-                built_info::TARGET,
-                built_info::BUILT_TIME_UTC,
-                built_info::RUSTC_VERSION,
+            let _tracing_guard = Tracing::builder()
+                .service_name("listener-operator")
+                .with_console_output((
+                    ENV_VAR_CONSOLE_LOG,
+                    LevelFilter::INFO,
+                    !telemetry_arguments.no_console_output,
+                ))
+                // NOTE (@NickLarsenNZ): Before stackable-telemetry was used, the log directory was
+                // set via an env: `LISTENER_OPERATOR_LOG_DIRECTORY`.
+                // See: https://github.com/stackabletech/operator-rs/blob/f035997fca85a54238c8de895389cc50b4d421e2/crates/stackable-operator/src/logging/mod.rs#L40
+                // Now it will be `ROLLING_LOGS` (or via `--rolling-logs <DIRECTORY>`).
+                .with_file_output(telemetry_arguments.rolling_logs.map(|log_directory| {
+                    let rotation_period = telemetry_arguments
+                        .rolling_logs_period
+                        .unwrap_or(RollingPeriod::Hourly)
+                        .deref()
+                        .clone();
+
+                    Settings::builder()
+                        .with_environment_variable(ENV_VAR_CONSOLE_LOG)
+                        .with_default_level(LevelFilter::INFO)
+                        .file_log_settings_builder(log_directory, "tracing-rs.log")
+                        .with_rotation_period(rotation_period)
+                        .build()
+                }))
+                .with_otlp_log_exporter((
+                    "OTLP_LOG",
+                    LevelFilter::DEBUG,
+                    telemetry_arguments.otlp_logs,
+                ))
+                .with_otlp_trace_exporter((
+                    "OTLP_TRACE",
+                    LevelFilter::DEBUG,
+                    telemetry_arguments.otlp_traces,
+                ))
```

Comment:
The whole point of `initialize_logging` was to insulate the individual operators from these details and instead standardize them across the platform.

Reply:
https://github.com/stackabletech/operator-rs/pull/1001 should resolve that concern. However, that comes after another breaking change. So we'd like to get this in so we unblock @labrenbe. There will be a follow up PR to turn this into a one-liner.

Reply:
Actually, this one isn't blocked by https://github.com/stackabletech/operator-rs/pull/1002, so as soon as https://github.com/stackabletech/operator-rs/pull/1001 is released we can implement it in this PR.

Reply:
Okay.

Reply:
Done in 7b8fa6aeeaf1e58f96de03fa3d1430f2da26a13c